internal/util/util.go

```diff
@@ -56,15 +70,44 @@ func ConnectNats(ctx context.Context, name string, srv string, tls TLSConfig, ol
 		opts = append(opts, nats.UseOldRequestStyle())
 	}
 
-	if tls != nil {
-		if tls.PrivateKey() != "" && tls.PublicCertificate() != "" {
+	var configuredTLS bool
+
+	if tlsc != nil {
+		if tlsc.PrivateKey() != "" && tlsc.PublicCertificate() != "" {
 			log.Infof("Configuring Client Certificate for connection")
-			opts = append(opts, nats.ClientCert(tls.PublicCertificate(), tls.PrivateKey()))
+			opts = append(opts, nats.ClientCert(tlsc.PublicCertificate(), tlsc.PrivateKey()))
+			configuredTLS = true
 		}
 
-		if tls.CertificateAuthority() != "" {
+		if tlsc.CertificateAuthority() != "" {
 			log.Infof("Configuring Certificate Authority for connection")
-			opts = append(opts, nats.RootCAs(tls.CertificateAuthority()))
+			opts = append(opts, nats.RootCAs(tlsc.CertificateAuthority()))
+		}
+	}
+
+	if choria != nil {
```

Comment:
@ploubser at present in this code base there's no way to test this and I dont really want to start pulling in brokers and stuff as dependencies just now.

A follow up PR will add it to the compose setup where it functions using this code.